Pod/Extensions/Visualizer+Singleton.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Visualizer+Singleton.swift
+//  TouchVisualizer
+//
+//  Created by Douglas Bumby on 2015-06-02.
+//  Copyright (c) 2015 molabo. All rights reserved.
+//
+
+import Foundation
+
+extension Visualizer {
+    public class func isEnabled() -> Bool {
+        return sharedInstance.enabled
+    }
+
+    // MARK: - Start and Stop functions
+    public class func start() {
+        start(Configuration())
+    }
+
+    public class func start(config: Configuration) {
+        let instance = sharedInstance
+        instance.enabled = true
+        instance.config = config
+
+        if let window = UIApplication.sharedApplication().keyWindow {
+            for subview in window.subviews {
+                if let subview = subview as? TouchView {
+                    subview.removeFromSuperview()
+                }
+            }
+        }
+    }
+
+    public class func stop() {
+        let instance = sharedInstance
+        instance.enabled = false
+
+        for touch in instance.touchViews {
+            touch.removeFromSuperview()
+        }
+    }
+
+    // MARK: - Dequeue and locating TouchViews and handling events
+    private func dequeueTouchView() -> TouchView {
+        var touchView: TouchView?
+        for view in touchViews {
+            if nil == view.superview {
```

Comment:
I think it is readable to locate easy to change value is left and not so easy to change value is right.